### PR TITLE
created checkboxes using material ui

### PR DESCRIPTION
**Pull request by manusankar9**

### CODE REVIEW

File: src/components/full-calender/components/index.js [hunk 1]
```diff
@@ -13,10 +14,6 @@ import interactionPlugin from '@fullcalendar/interaction';
 export default class EventCalender extends Component {
     date;
 
-    myRef = React.createRef();
-
-    isEmpty = value =>!(/^\s*$/.test(value)) && !(value === null);
-
     handleEvents = (e) => {
         this.props.allNewCalenderEvents({ label: 'e.target.label', ischecked: e.target.checked, ObjId: e.target.value });
```
Comment (sivasankar9) [inline src/components/full-calender/components/index.js]: remove the unecessay code i.e  label

Comment (sivasankar9) [inline src/components/full-calender/components/index.js]: need s' to be destructed = e.target

**Review by sivasankar9 — CHANGES_REQUESTED** [review]

File: src/components/full-calender/components/index.js (hunk 1 above)
Comment (manusankar9) [reply src/components/full-calender/components/index.js]: fixed issue

Comment (manusankar9) [reply src/components/full-calender/components/index.js]: fixed issue

Comment (manusankar9) [reply src/components/full-calender/components/index.js]: @sivasankar9 fixed issue

Comment (manusankar9) [reply src/components/full-calender/components/index.js]: @sivasankar9 
fixed issue

**Pull request merged**